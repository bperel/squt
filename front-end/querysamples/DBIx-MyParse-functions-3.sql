SELECT TRIM(LEADING 'x' FROM 'xxxbarxxx')
