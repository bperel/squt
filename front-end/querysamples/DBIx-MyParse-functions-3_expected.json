{
  "Warning": {
    "No alias": {
      "ltrim": ["function", "SELECT"]
    }
  },
  "Functions": {
    "0": {
      "to": "OUTPUT",
      "Constants": {
        "x": "x",
        "xxxbarxxx": "xxxbarxxx"
      },
      "name": "ltrim",
      "alias": "0"
    }
  }
}
